app/classes/Stores/Project.php

```diff
@@ -140,8 +140,8 @@ class Project
         'google' => [
             // channel => path to template file
             'release' => [
-                'template' => 'google/release/listing_nov_2014.php',
-                'langfile' => 'description_page.lang',
+                'template' => 'google/release/listing_oct_2015.php',
+                'langfile' => ['android_42_release.lang'],
```

Comment:
Why not add an additional 'whatsnew' field? That's probably the route I would have chosen